metaflow/plugins/aws/batch/batch_decorator.py

```diff
@@ -27,35 +29,6 @@
     from urllib.parse import urlparse
 
 
-class ResourcesDecorator(StepDecorator):
```

Comment:
Since we're aiming to contribute this back upstream, I don't think we should delete any existing code in batch. Since this class doesn't conflict with any code you've written, I would add this back.